shell/lib/db-deprecated.js

```diff
@@ -65,15 +65,43 @@ if (Meteor.isServer) {
 }
 
 if (Meteor.isClient) {
-  globalTopbar = new SandstormTopbar({
-    get: function () {
-      return Session.get("topbar-expanded");
+  Session.setDefault("show-sidebar", true);
+  FancyReactiveVar = function(initialValue) {
+    // Hack: ReactiveVar implementation doesn't preserve the original object prototype, which
+    // breaks reactivity.
+    this._val = (initialValue === undefined) ? [] : initialValue;
+    this._dep = new Tracker.Dependency;
+    this.get = function() {
+      this._dep.depend();
+      return this._val;
     },
-
-    set: function (value) {
-      Session.set("topbar-expanded", value);
+    this.set = function(value) {
+      this._val = value;
+      this._dep.changed();
     }
-  });
+    return this;
+  };
+  globalGrains = new FancyReactiveVar([]);
```

Comment:
It'd be nice to have some documentation about the type of this variable. An array of ... what?